### ISSUE TITLE
gap between software requirements and documentation for ECE

### ISSUE THREAD

**Issue by 111andre111** [elastic/ansible-elastic-cloud-enterprise#135]
Let me describe with a table because that should illustrate better what I mean.
Basically in the documentation we sometimes mention another concrete software than in the actual documentation

| OS  | Documentation | Ansible file | Docker/Cli Doc vs Docker/Cli Ansible | Containerd Doc vs Containerd Ansible |
|----|----------------|-------------|------------------------------|----------------------------------------|
| CentOS 7 | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-rhel-centos-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_CentOS_7.yml | 20.10.8 vs 20.10.* but later than 20.10.7 | 1.4.3 vs 1.4.* |
| CentOS 8 | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-rhel-centos-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_CentOS_8.yml | 20.10.8 vs 20.10.* but later than 20.10.7 | 1.4.3 vs 1.4.* |
| RHEL 7 | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-rhel-centos-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_RedHat_7.yml | 20.10.8 vs 20.10.* but later than 20.10.7 | 1.4.3 vs 1.4.* |
| RHEL 8 | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-rhel-centos-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_RedHat_8.yml | 20.10.8 vs 20.10.* but later than 20.10.7 | 1.4.3 vs 1.4.* |
| Ubuntu 16 | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-ubuntu-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_Ubuntu_16.yml | 19.03.15* vs 19.03.* | 1.4.3-1* vs 1.2.* |
| Ubuntu 18 | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-ubuntu-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_Ubuntu_18.yml | 19.03.15* vs 19.03.* | 1.4.3-1* vs 1.2.* |
| SLES 12 (ok) | https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-hosts-sles12-cloud.html | https://github.com/elastic/ansible-elastic-cloud-enterprise/blob/master/vars/os_SLES_12.yml | docker-19.03.14_ce vs docker-19.03.14_ce | - |

So to summarize that is seems that there is only SLES12 matching the requirements at the moment.
